src/Control/Monad/Free/TH.hs

```diff
@@ -3,7 +3,11 @@
 {-# OPTIONS_GHC -Wno-overlapping-patterns #-}
 #endif
 #include "free-common.h"
```

Comment:
Nitpick: I'd prefer `#include "free-common.h"` to come after all of the language pragmas, for consistency with the rest of the library.

Reply:
Done